generators/utilities/src/main/resources/csrc/emulator.cc

```diff
@@ -160,6 +163,8 @@ int main(int argc, char** argv)
       case 'r': rbb_port = atoi(optarg);    break;
       case 'V': verbose = true;             break;
       case 'D': dramsim = 1;                break;
+      case 'p': opterr = 0;                 break;
```

Comment:
why not just call this variable `permissive`? This isn't telling you that there was an options error right, just masking them?

Reply:
I mirrored the variable name in htif/fesvr but I can name it whatever.

Reply:
I'll change this after the current CI run finishes in case I need to fix anything else.

Reply:
Oh, I see. I mean that's a fine reason to leave it as-is then; do whichever you like better